src/Sarif/CodeGenHints.json

```diff
@@ -45,6 +45,11 @@
       "$type": "Microsoft.Json.Schema.ToDotNet.DictionaryHint, Microsoft.Json.Schema.ToDotNet"
     }
   ],
+  "Run.LogicalLocations": [
+    {
+      "$type": "Microsoft.Json.Schema.ToDotNet.DictionaryHint, Microsoft.Json.Schema.ToDotNet"
```

Comment:
Not your bug, but can you remove 'valueTypeName' from Run.Rules below? I don't think it's consumed (i added it speculatively and accidentally checked it in).